src/test/java/test/utils/repository/standard/PersonRepository.java

```diff
@@ -21,6 +21,7 @@
 import org.springframework.data.domain.Slice;
 import org.springframework.data.domain.Sort;
 import org.springframework.data.hazelcast.repository.HazelcastRepository;
+import org.springframework.data.hazelcast.repository.config.EnableHazelcastRepositoriesIT;
```

Comment:
Why is this import needed if nothing changed in the class itself?

Reply:
There is java doc which refers to this class: https://github.com/dineshbhagat/spring-data-hazelcast/blob/master/src/test/java/test/utils/repository/standard/PersonRepository.java#L40

Reply:
Let me know, if that is not required, will remove it.

Reply:
Oh, ok - it's just for the sake of JavaDoc then? It's ok then.